docs/architecture/grpc-for-wcf-developers/network-protocols.md

```diff
@@ -1,11 +1,35 @@
 ---
 title: Network protocols - gRPC for WCF Developers
-description: TO BE WRITTEN
+description: An overview of the gRPC network protocols.
 author: markrendle
 ms.date: 09/02/2019
 ---
 
 # Network protocols
 
+Unlike WCF, gRPC uses HTTP/2 as a base for its networking. This offers significant advantages over WCF and SOAP, which only operate on HTTP/1.1. For developers wanting to use gRPC, given that there's no alternative to HTTP/2, it would seem to be the ideal moment to explore HTTP/2 in more detail and identify additional benefits to using gRPC.
+
+HTTP/2, released by Internet Engineering Task Force in 2015, was derived from the experimental SPDY protocol, which was already being used by Google. It was specifically designed to be simpler, faster, and more secure than HTTP/1.1.
```

Comment:
Is HTTP/2 really simpler than HTTP/1? I now know a lot about HTTP/2 and it is pretty complicated 😋

Reply:
Yea not simpler but more efficient.